tests/test_core.py

```diff
@@ -6973,6 +6973,8 @@ def test_ccall(self):
       var multi = Module['cwrap']('multi', 'number', ['number', 'number', 'number', 'string']);
       out(multi(2, 1.4, 3, 'atr'));
       out(multi(8, 5.4, 4, 'bret'));
+      var printBool = Module['cwrap']('print_bool', null, ['boolean']);
+      out(Module['_print_bool'] === printBool); // the function should be the exact raw function in the module rather than a wrapped one
```

Comment:
I think this check is the problem.  The exact match is an optimization we do in release mode.  It debug builds you always get the wrapper function.   I guess you could wrap this is some kind of `#if` block..

Reply:
I've moved this check to another test function, and manged to make it pass in my local environment.